yeast-markdown-parser/src/__tests__/tests/MarkdownParser.test.ts

```diff
@@ -491,6 +510,7 @@ test('MarkdownParser parses hyperlink in text', () => {
 
 	// Parse
 	const ast = parser.parse(LINK_MARKDOWN);
+	// debugAST('links', ast);
```

Comment:
I checked in these debug statements as comments on purpose. This is super helpful in debugging tests when running locally and makes it easy for the developer to dump the parsed AST.